projects/convai2/eval_ppl.py

```diff
@@ -151,6 +151,7 @@ def parley(self):
 
         # hide labels from model
         labels = action.pop('eval_labels', action.pop('labels', None))
+        action.pop('label_candidates')
```

Comment:
If you add “, None” after label candidates then there wont be an error if label candidates arent there